dbt_copilot_helper/commands/check_cloudformation.py

```diff
@@ -49,6 +50,25 @@ def lint(ctx: click.Context, directory: str) -> bool:
     return success
 
 
+@check_cloudformation.command()
+@click.option("-d", "--directory", type=str, default="copilot")
+@click.pass_context
+def static(ctx: click.Context, directory: str) -> bool:
+    addons_manifests = f"{directory}/**/addons/*.yml"
```

Comment:
Would be nice to have a description for the documentation, and I think we should inclue the `overrides` directories too...
```suggestion
    """Scans generated CloudFormation templates for security issues."""
    addons_manifests = f"{directory}/**/addons/*.yml"
    addons_manifests = f"{directory}/**/overrides/*.yml"
```

Reply:
@WillGibson cfn.patches.yml isn't a cloudformation template itself though is it? The `lint` command doesn't check it

Reply:
You're right, they are just YAML patches. Ignore this comment.